scale-encode/src/error/mod.rs

```diff
@@ -156,16 +165,37 @@ pub enum Kind {
     Number,
 }
 
+impl Display for Kind {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "{self:?}")
```

Comment:
With the above suggestion we won't need this impl, which feels a bit pointless anyway as it's just the debug one :)

Reply:
Removed ❤️